src/legacy/core_plugins/kibana/public/visualize/wizard/type_selection/new_vis_help.tsx

```diff
@@ -30,5 +30,16 @@ export const NewVisHelp = () => (
         defaultMessage="Start creating your visualization by selecting a type for that visualization."
       />
     </p>
+    <p>
+      <strong>
+        <FormattedMessage
+          id="kbn.visualize.newVisWizard.tryLensText"
+          defaultMessage="Try Lens, our new, intuitive way to create visualizations."
+        />
+      </strong>
+    </p>
+    <EuiButton fill size="s" iconType="popout" iconSide="right">
+      Go to Lens
+    </EuiButton>
```

Comment:
You're adding this to OSS code, which won't make sense in all cases Lens is not OSS. I think @chrisdavies might need to add to the visualization registry to support this